docs/DataProviders.md

```diff
@@ -88,6 +88,7 @@ Developers from the react-admin community have open-sourced Data Providers for m
 * **[Sails.js](https://sailsjs.com/)**: [mpampin/ra-data-json-sails](https://github.com/mpampin/ra-data-json-sails)
 * **[Spring Boot](https://spring.io/projects/spring-boot)**: [vishpat/ra-data-springboot-rest](https://github.com/vishpat/ra-data-springboot-rest) 
 * **[Strapi](https://strapi.io/)**: [nazirov91/ra-strapi-rest](https://github.com/nazirov91/ra-strapi-rest)
+* **[Express & Sequelize](https://github.com/lalalilo/express-sequelize-crud)**: [express-sequelize-crud](https://github.com/lalalilo/express-sequelize-crud)
```

Comment:
We list data providers by alphabetical order. Would you mind adding yours between D and F?

Reply:
Oups. Sure. Damn it, I should have found a name starting with an `A`